components/breadcrumb/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@vonage/vwc-breadcrumb",
+	"version": "2.26.0",
+	"description": "breadcrumb component",
+	"homepage": "https://github.com/Vonage/vivid/tree/master/components/breadcrumb#readme",
+	"license": "ISC",
+	"main": "vwc-breadcrumb.js",
+	"module": "vwc-breadcrumb.js",
+	"files": [
+		"*.js",
+		"*.ts",
+		"*.map"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/vonage/vivid.git",
+		"directory": "components/breadcrumb"
+	},
+	"scripts": {
+		"build:typescript": "tsc -b",
+		"build:styles": "umbrella-style-modules",
+		"build": "yarn run build:styles && yarn run build:typescript"
+	},
+	"bugs": {
+		"url": "https://github.com/Vonage/vivid/issues"
+	},
+	"dependencies": {
+		"@vonage/vivid": "^3.0.0-next.2",
```

Comment:
Won't it conflict?

Reply:
Conflict with what?

Reply:
Consumers installing @vonage/vivid `< 3` and this

Reply:
Nope - because the import syntax from `vivid-3` is totally different than what ppl will do in `vivid-2` :)